docs/src/stream.rst

```diff
@@ -133,6 +133,17 @@ API
     .. note::
         `server` and `client` must be handles running on the same loop.
 
+.. c:function:: int uv_accept_reject(uv_stream_t* server)
```

Comment:
Perhaps we can just call this `uv_reject`?

Also, please add a version added indicator.